safe/note.go

```diff
@@ -107,7 +107,10 @@ func NewEmptyNote(name string) note {
 }
 
 func editorReadText(existingText string) (string, error) {
-	editor := os.Getenv("EDITOR")
+	editor := os.Getenv("VISUAL")
+	if editor == "" {
+		editor = os.Getenv("EDITOR")
+	}
 	if editor == "" {
```

Comment:
This `if` statement can move inside the `if` statement above. Might be a good idea to move this logic to a new function, e.g.
```go
func editor() string {
    // editor := os.Getenv..
    return editor
}
```
or even
```go
func editorPath() (string, error) // also does the exec.LookPath(editor) lookup
```